PSR2R/Sniffs/Classes/BraceOnSameLineSniff.php

```diff
@@ -2,32 +2,22 @@
 
 namespace PSR2R\Sniffs\Classes;
 
-use PHP_CodeSniffer_File;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
 
 /**
  * Makes sure opening braces are on the same line for class, interface and trait.
  *
- * @author Mark Scherer
+ * @author  Mark Scherer
```

Comment:
PhpStorm "Reformat Code" does that. I need to see if there's a setting I can adjust. There are likely many places that way. I agree, it goes against the PSR2R standard/recommendations.